models/evc.py

```diff
@@ -749,10 +749,12 @@ def _prepare_direct_uni_flows(self):
         vlan_z = self.uni_z.user_tag.value if self.uni_z.user_tag else None
 
         flow_mod_az = self._prepare_flow_mod(
-            self.uni_a.interface, self.uni_z.interface, self.queue_id
+            self.uni_a.interface, self.uni_z.interface,
+            self.queue_id, bool(vlan_a)
```

Comment:
The `bool` transformation will work, currently, available vlans on interfaces are from [1 to 4095](https://github.com/kytos-ng/kytos/blob/master/kytos/core/interface.py#L104), vlan 0 also is meant to be reserved, however to avoid potential corner cases maybe we could could check if it's not None?

Reply:
Yes, having a 0 will result in a `False` value. Changed